stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -1349,10 +1349,10 @@ public func _setDownCast<BaseValue, DerivedValue>(_ source: Set<BaseValue>)
   && _isClassOrObjCExistential(DerivedValue.self) {
     switch source._variantStorage {
     case _VariantSetStorage.native(let storage):
-      let bridgingStorage = storage.bridgingStorage()
+      let bridged = storage.bridged()
```

Comment:
This constant doesn't seem to help the code; I'd just fold it into the expression below.